generators/languages/types.d.ts

```diff
@@ -38,6 +40,15 @@ export type Options = BaseApplicationOptions & Omit<Command['Options'], 'languag
 
 export type Source = BaseApplicationSource & {
   addEntityTranslationKey: (arg: { translationKey: string; translationValue: string; language: string }) => void;
+  addLanguages: ({ languagesDefinition }: { languagesDefinition: readonly Language[] }) => void;
+  updateLanguagesClient: (
+    gen: BaseApplicationGenerator<any, any, any, any, any, any, any>,
+    { application, control, source }: { application: Application<any>; control: Control; source: Source },
+  ) => void;
+  updateLanguagesServer: (
+    gen: BaseApplicationGenerator<any, any, any, any, any, any, any>,
+    { application, control, source }: { application: Application<any>; control: Control; source: Source },
+  ) => void;
```

Comment:
source api should be simple.
Is very bad.

Reply:
Totally agrees that only one api to trigger them all would be the graal.
But didn't want to modify to business code of something that worked.
Can we agree on doing the thing two step (1. moving the code where it should be, 2. improving the API)? I can add a comment marking as debt with the action to achieve as remediation if you agrees